src/diffusers/schedulers/scheduling_ddim_flax.py

```diff
@@ -155,9 +161,17 @@ def set_timesteps(
                 the `FlaxDDIMScheduler` state data class instance.
             num_inference_steps (`int`):
                 the number of diffusion steps used when generating samples with a pre-trained model.
-            offset (`int`):
-                optional value to shift timestep values up by. A value of 1 is used in stable diffusion for inference.
         """
+        offset = self.config.steps_offset
+        if "offset" in kwargs:
+            warnings.warn(
```

Comment:
Let's not even add this warning as we've never released Flax schedulers with the "old/wrong" API